lxc/action.go

```diff
@@ -231,30 +290,13 @@ func (c *cmdAction) Run(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		for _, resource := range resources {
```

Comment:
So one thing to keep in mind too is that the CLI needs to be backward compatible.

So you'll instead want to keep the code as it is and add a short path like:
```
if c.flagAll && d.HasExtension("name_of_extension") {
    return doActionAll(...)
}
```